### ISSUE TITLE
Change text from "Related Topics" to "Related tags"

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#7980]
### Please describe the idea

Over on the curatorial side of PublicLab.org, we are clarifying the use of "tag" vs "Topic". Many, many things can be tags, but only a few are Topics. For this reason, we are requesting that on tag pages, for instance, https://publiclab.org/tag/water-quality, the right hand sidebar text that says "Related Topics" be changed to "related tags".

![Screen Shot 2020-05-20 at 2 41 53 PM](https://user-images.githubusercontent.com/161439/83684490-1a78a800-a5b5-11ea-8c47-cdc3536baf52.png)

> What did you expect to see that you didn't?

I expected the right hand sidebar of tag pages to display "Related tags" above the list of tags. 

### Please show us where to look

https://publiclab.org/tag/water-quality

### What's your PublicLab.org username?
liz

Thanks!

**Comment by Tlazypanda** [comment]
I think we can make a fto out of this as well @ebarry What do you think? :sweat_smile:

**Comment by Tlazypanda** [comment]
Hey @ebarry ! Looks like we have some new contributors looking for ftos :sweat_smile:

**Comment by ebarry** [comment]
Yes, it would be wonderful to make this into an FTO!!! thank you @Tlazypanda sorry i was slow to respond

**Comment by Tlazypanda** [comment]
@ebarry Just created the fto here https://github.com/publiclab/plots2/issues/8004 :smile:  And you must have been busy totally understand that!! :sweat_smile:
